runners/core-java/src/test/java/org/apache/beam/runners/core/SplittableParDoTest.java

```diff
@@ -146,18 +151,23 @@ public void testBoundednessForUnboundedFn() {
         "Applying an unbounded SDF to a bounded collection produces a bounded collection",
         PCollection.IsBounded.UNBOUNDED,
         makeBoundedCollection(pipeline)
-            .apply("unbounded to bounded", new SplittableParDo<>(unboundedFn))
+            .apply("unbounded to bounded", new SplittableParDo.Bound<>(ParDo.of(unboundedFn)))
             .isBounded());
     assertEquals(
         "Applying an unbounded SDF to an unbounded collection produces an unbounded collection",
         PCollection.IsBounded.UNBOUNDED,
         makeUnboundedCollection(pipeline)
-            .apply("unbounded to unbounded", new SplittableParDo<>(unboundedFn))
+            .apply("unbounded to unbounded", new SplittableParDo.Bound<>(ParDo.of(unboundedFn)))
```

Comment:
Referring to my prior comments, all the things `set` just below should be passed to this constructor.